ui/components/req-filter-ui.jsx

```diff
@@ -0,0 +1,71 @@
+import axios from 'axios';
+import React from 'react';
+import { Link } from 'react-router';
+
+import { apiUrl } from '../globals';
+
+function FilterItem({ keywordIds, keyword, query }) {
```

Comment:
Can do! Thanks for the pointer, too; I've been avoiding classes so far, but inheritance is exactly where they should shine.